packages/@aws-cdk/aws-ssm/test/test.parameter.ts

```diff
@@ -128,12 +129,15 @@ export = {
     test.done();
   },
 
-  'StringParameter.fromName'(test: Test) {
+  'StringParameter.fromStringParameterAttributes'(test: Test) {
     // GIVEN
     const stack = new Stack();
 
     // WHEN
-    const param = ssm.StringParameter.fromStringParameterName(stack, 'MyParamName', 'MyParamName');
+    const param = ssm.StringParameter.fromStringParameterAttributes(stack, 'MyParamName', {
```

Comment:
Ehrmagerd the length of this function call :(

Reply:
yak I know